services/common/boltzmann/client/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const fetch = require('node-fetch')
```

Comment:
You might want to use the wrapped `fetch` that was merged in #177. #183 is still open that aims to do some refactoring of the original merge

Reply:
Ah, this is on the server side!